src/actions/sso/index.ts

```diff
@@ -27,9 +27,23 @@ export const clearCredentialRequest = () => {
 
 export const parseJWT = (encodedJwt: string) =>{
   return async(dispatch: Dispatch<AnyAction>, getState: Function, backendMiddleware: BackendMiddleware) => {
+    const { storageLib } = backendMiddleware
```

Comment:
What is the purpose of this?

Reply:
Do you mean the purpose of using `{ }`? It can be removed as there's just one param